tests/cli_test.ts

```diff
@@ -398,3 +398,114 @@ Deno.test("fresh-init .", async function (t) {
     assertEquals(code, 0);
   });
 });
+
+Deno.test({
+  name: "fresh-init subdirectory",
+  async fn(t) {
+    // Preparation
+    const tmpDirName = await Deno.makeTempDir();
+
+    await t.step("execute init command", async () => {
+      await Deno.mkdir(path.join(tmpDirName, "subdirectory"));
+
+      const cliProcess = new Deno.Command(Deno.execPath(), {
+        args: [
+          "run",
+          "-A",
+          path.join(Deno.cwd(), "init.ts"),
+          "subdirectory/subsubdirectory",
+        ],
+        cwd: tmpDirName,
+        stdin: "null",
+        stdout: "piped",
+        stderr: "piped",
+      });
+      const { code } = await cliProcess.output();
+      assertEquals(code, 0);
+    });
+
+    await t.step(
+      "move deno.json and import_map.json one level up",
+      async () => {
```

Comment:
This doesn't have to happen in a test step. It's not testing anything. Can you move both this, and the previous "execute init command" out of the `t.step` (one indent up)?

Reply:
I did not notice this. I don't really understand what `t.step` is, but I think I get it now. Thanks!